[2023-06-17T02:27:00.585+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_dag.executor_task scheduled__2023-06-16T21:00:00+00:00 [queued]>
[2023-06-17T02:27:00.682+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_dag.executor_task scheduled__2023-06-16T21:00:00+00:00 [queued]>
[2023-06-17T02:27:00.682+0000] {taskinstance.py:1308} INFO - Starting attempt 4 of 4
[2023-06-17T02:27:00.777+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): executor_task> on 2023-06-16 21:00:00+00:00
[2023-06-17T02:27:00.786+0000] {standard_task_runner.py:57} INFO - Started process 3815 to run task
[2023-06-17T02:27:00.790+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'demo_dag', 'executor_task', 'scheduled__2023-06-16T21:00:00+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/demo_dag.py', '--cfg-path', '/tmp/tmptqcty1b5']
[2023-06-17T02:27:00.793+0000] {standard_task_runner.py:85} INFO - Job 115: Subtask executor_task
[2023-06-17T02:27:00.930+0000] {task_command.py:410} INFO - Running <TaskInstance: demo_dag.executor_task scheduled__2023-06-16T21:00:00+00:00 [running]> on host 69b08131bb85
[2023-06-17T02:27:01.548+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-06-17T02:27:01.600+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='personaltraingcp@gmail.com' AIRFLOW_CTX_DAG_OWNER='Leyner_Bejarano' AIRFLOW_CTX_DAG_ID='demo_dag' AIRFLOW_CTX_TASK_ID='executor_task' AIRFLOW_CTX_EXECUTION_DATE='2023-06-16T21:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-16T21:00:00+00:00'
[2023-06-17T02:27:02.053+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-17T02:27:02.054+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', ' \n        cd /opt/***/dags/scripts &&\n        pip install -r requirements.txt &&\n        python lambdaExecutor.py -dt 2023-06-16 -dh 21 -q https://sqs.us-east-2.amazonaws.com/147929168898/lokaInterviewSQS -ak AKIASE4KD7QBGVHE4G7U -sk *** &&\n        pip freeze\n        ']
[2023-06-17T02:27:02.274+0000] {subprocess.py:86} INFO - Output:
[2023-06-17T02:27:43.119+0000] {local_task_job_runner.py:292} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2023-06-17T02:27:43.121+0000] {process_utils.py:135} INFO - Sending Signals.SIGTERM to group 3815. PIDs of all processes in the group: [3816, 3817, 3815]
[2023-06-17T02:27:43.121+0000] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 3815
[2023-06-17T02:27:43.122+0000] {taskinstance.py:1517} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-06-17T02:27:43.122+0000] {subprocess.py:104} INFO - Sending SIGTERM signal to process group
[2023-06-17T02:27:43.254+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=3815, status='terminated', exitcode=0, started='02:27:00') (3815) terminated with exit code 0
[2023-06-17T02:27:43.254+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=3816, status='terminated', started='02:27:01') (3816) terminated with exit code None
[2023-06-17T02:27:43.254+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=3817, status='terminated', started='02:27:03') (3817) terminated with exit code None
